### DISCUSSION TITLE
Release of the wagtail v6.0 compatibility

### DISCUSSION THREAD

**Discussion by ho-tsl** [wagtail/wagtail-localize#780] [General]
Hey there,

I have an environment built with Wagtail v6.0.1. When I access a translated page, the layout of the footer dropdown navigation appears as follows:
![Screenshot_9](https://github.com/wagtail/wagtail-localize/assets/158229836/8879c835-4c76-44fa-a855-c960fe94776c)

I was wondering if you have already planned the release date for the version that supports Wagtail v6.0.

**Comment by zerolab** [comment]
I've released https://github.com/wagtail/wagtail-localize/releases/tag/v1.9-alpha.1 which should solve that. I'd appreciate some tests

**Comment by ho-tsl** [comment]
@zerolab , The layout issue has been resolved, but I'm encountering an issue with publishing translated pages. After a prolonged loading, the process abruptly stops loading, and the page is not published. Please refer to the following screenshot:
![image](https://github.com/wagtail/wagtail-localize/assets/158229836/77331d19-25e2-4e15-9f68-63b514973857)

I was also attempting to use the wagtail-localize[google] package, but I encountered difficulties in downloading its pip dependencies along with the alpha version of wagtail-localize. Consequently, I was unable to test it.

**Reply by zerolab** [reply]
can you check your browser error console? are there any JavaScript errors?

**Reply by ho-tsl** [reply]
It's empty. Not even a network request was sent.

**Reply by ho-tsl** [reply]
Installed it like that:
![image](https://github.com/wagtail/wagtail-localize/assets/158229836/0e3d9aff-53d0-4c37-935c-c1f705c5ea3e)

**Reply by zerolab** [reply]
Will have a look tomorrow 🤔

**Reply by ho-tsl** [reply]
Hey @zerolab , any updates?

**Reply by zerolab** [reply]
Found the issue - https://github.com/wagtail/wagtail-localize/pull/782

Will make a new release shortly

**Reply by zerolab** [reply]
https://github.com/wagtail/wagtail-localize/releases/tag/v1.9-alpha.2

**Reply by ho-tsl** [reply]
@zerolab , it does the request now. However, it is not publishing. It's not returning errors either. Checking the network console, the request it triggers is a GET request, so basically it's just reloading the page.

**Reply by zerolab** [reply]
https://pypi.org/project/wagtail-localize/1.9a3/ should fix that

**Reply by ho-tsl** [reply]
@zerolab , it's all working now. Cool. Thank you.

Related to my tests, though, the layout of the buttons seems a bit weird now. I'm just reporting because you asked for tests, but this doesn't annoy me at all, hahah.
![image](https://github.com/wagtail/wagtail-localize/assets/158229836/384c81f2-acd1-4b43-b8a1-5455cf129f15)

One more thing... Do you have any idea of when this release would be launched?

**Reply by zerolab** [reply]
huh, I did not notice the button issues 🤦🏼 

If all goes well, I'll probably make a release on Friday

**Comment by zerolab** [comment]
v1.9 should have solved all rthe raised issues
